src_c/event.c

```diff
@@ -598,12 +594,22 @@ _pg_pgevent_deproxify(Uint32 type)
 #if IS_SDLv2
 static SDL_Event *_pg_last_keydown_event = NULL;
 
+static int
+_pg_translate_windowevent(void *_, SDL_Event *event)
+{
+    if (event->type == SDL_WINDOWEVENT) {
+        event->type = PGE_WINDOWSHOWN + event->window.event - 1;
+        return SDL_EventState(_pg_pgevent_proxify(event->type), SDL_QUERY);
```

Comment:
What's going on here?

Reply:
That’s where we do SDL_WINDOWEVENT translation into respective events. I figured out the old approach to do that from the event filter has its disadvantages, so we do it here now